### PR TITLE
Remove extra brackets in develop

### PR DESCRIPTION
**Pull request by harriswong**
## Description
Remove the extra brackets on the `require_once` line for `classes/class-wc-rest-connect-service-data-refresh-controller.php`. Also ran `vendor/bin/phpcbf classes/* tests/php/ woocommerce-services.php` for phpcs linting so that [this PR](https://github.com/Automattic/woocommerce-services/pull/2322)'s effort is kept.

### Related issue(s)
Closes https://github.com/Automattic/woocommerce-services/issues/2349

### Steps to reproduce & screenshots/GIFs
1. First commit is to remove the bracket
2. Second commit is `vendor/bin/phpcbf classes/* tests/php/ woocommerce-services.php`
3. Run PHPUnit and tests should pass.

### Checklist

- [ ] unit tests

- [ ] `changelog.txt` entry added

### CODE REVIEW

**Review by frosso — APPROVED** [review]
Whoopsie, thanks!

File: woocommerce-services.php [hunk 1]
```diff
@@ -843,7 +843,7 @@ public function rest_api_init() {
 			$this->set_rest_self_help_controller( $rest_self_help_controller );
 			$rest_self_help_controller->register_routes();
 
-			require_once __DIR__ . 'classes/class-wc-rest-connect-service-data-refresh-controller.php' ) );
+			require_once __DIR__ . 'classes/class-wc-rest-connect-service-data-refresh-controller.php';
```
Comment (waclawjacek) [inline woocommerce-services.php:846]: There seems to be a leading slash missing here:

```suggestion
			require_once __DIR__ . '/classes/class-wc-rest-connect-service-data-refresh-controller.php';
```

**Review by waclawjacek — CHANGES_REQUESTED** [review]

File: woocommerce-services.php (hunk 1 above)
Comment (frosso) [reply woocommerce-services.php:846]: 🦅 👀 
I really messed up that merge conglict, huh? :D

Comment (harriswong) [reply woocommerce-services.php]: Good catch! Thanks!! Fixed.

**Review by waclawjacek — APPROVED** [review]
LGTM! 🚢

**Pull request merged**